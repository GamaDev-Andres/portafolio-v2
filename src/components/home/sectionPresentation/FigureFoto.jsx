import miFoto from '../../../assets/img/foto_yo.jpg'

const FigureFoto = () => {
  return (
    <figure className="aspect-square p-4 max-w-xs mx-auto rounded-full">
      <img
        className="w-full object-cover object-top  shadow-md shadow-blue-100 h-full rounded-full"
        src={miFoto}
        alt="foto Andres Gama"
      />
    </figure>
  )
}

export default FigureFoto
