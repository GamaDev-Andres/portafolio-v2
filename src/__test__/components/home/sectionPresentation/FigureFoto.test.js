/**
 * @jest-environment jsdom
 */
import Enzyme from 'enzyme'
import FigureFoto from '../../../../components/home/sectionPresentation/FigureFoto'

describe('probando <FigureFoto/>', () => {
  const wrapper = Enzyme.mount(
    <FigureFoto />
  )
  test('debe montarse correctamente', () => {
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('figure').length).toBe(1)
  })
})
