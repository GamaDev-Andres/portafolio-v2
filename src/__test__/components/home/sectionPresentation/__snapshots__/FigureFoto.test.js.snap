// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`probando <FigureFoto/> debe montarse correctamente 1`] = `
<figure
  className="aspect-square p-4 max-w-xs mx-auto rounded-full"
>
  <img
    alt="foto Andres Gama"
    className="w-full object-cover object-top  shadow-md shadow-blue-100 h-full rounded-full"
    src={Object {}}
  />
</figure>
`;
